iocage/iocage.8

```diff
@@ -0,0 +1,1646 @@
+.El
+.\" == PKG ==
+.It Cm pkg
+Run desired
+.Cm pkg
+commands in the specified jail.
+.\" == RESTART ==
+.It Cm restart
+Restart the specified jail.
+Use ALL to restart all jails.
+.Pp
+Options:
+.Bl -tag -width "[-s | --soft]"
+.It Op Fl -soft | s
+Restart the jail, but do not tear down the network stack.
+.El
+.\" == ROLLBACK ==
+.It Cm rollback
+Roll back a jail to an existing snapshot.
+Any intermediate snapshots are destroyed in the process.<REVIEW>
+For more information on this functionality, please see
+.Xr zfs 8 .
+.Pp
+Options:
+.Bl -tag -width "[-f | --force]"
+.It Op Fl -force | f
+Run the command, skipping any warnings or further user interaction.
+.It Op Fl -name | n
+[Required] Used to specify the snapshot name.
+.El
+.\" == SET ==
+.It Cm set
+Set the specified property in the desired jail.
+.Pp
+Options:
+.Bl -tag -width "[-P | --plugin]"
+.It Op Fl -plugin | P
+Set the specified key for a plugin jail.
+If accessing a nested key, use "." as a separator.
+.El
+.Pp
+Example:
+.Bl -item
+.It
+.Nm
+set -P foo.bar.baz=VALUE PLUGIN
+.El
+.\" == SNAPLIST ==
+.It Cm snaplist
+List snapshots of a jail.
+A number of different fields are displayed:
+.Pp
+.Bl -tag -width "CREATED" -compact -offset indent
+.It NAME
+snapshot name
+.It CREATED
+creation time
+.It RSIZE
+referenced size
+.It USED
+used space
+.El
+.Pp
+Options:
+.Bl -tag -width "[-H | -h | --delete]"
+.It Op Fl -header | H | h
+Used for scripting.
+Tabs are used as separators.
+.El
+.\" == SNAPREMOVE ==
+.It Cm snapremove
+Delete snapshots of the specified jail.
+If the keyword
+.Ar ALL
+is used, all snapshots the specified jail are deleted.
+.Pp
+Options:
+.Bl -tag -width "[-n | --name]"
+.It Op Fl -name | n
+[Required] The snapshot name <REVIEW>.
+.El
+.\" == SNAPSHOT ==
+.It Cm snapshot
+Create a ZFS snapshot of the specified jail.
+If a snapshot name is not specified, a name based on the current
+date and time is generated.
+.Pp
+Options:
+.Bl -tag -width "[-n | --name]"
+.It Op Fl -name | n
+The user created snapshot name.
+.El
+.\" == START ==
+.It Cm start
+Start a jail identified by
+.Ar UUID
+or
+.Ar TAG
+ . Use ALL to start all installed jails.
+.Pp
+Options:
+.Bl -tag -width "[--rc]"
+.It Op Fl -rc
+Start all jails with boot=on in a specific order.
+Jails with lower priority start first <REVIEW>.
+.El
+.\" == STOP ==
+.It Cm stop
+Stop a jail identified by
+.Ar UUID
+or
+.Ar TAG
+ . Use ALL to stop all active jails.
+.Pp
+Options:
+.Bl -tag -width "[--rc]"
+.It Op Fl -rc
+Stop all jails with boot=on in a specific order.
+Jails with higher priority values stop first <REVIEW>.
+.El
+.\" == UPDATE ==
+.It Cm update
+Run
+.Cm freebsd-update
+to update the specified jail to the latest patch level.
+A backup snapshot is automatically created to provide a rollback
+option.
+.\" == UPGRADE ==
+.It Cm upgrade
+Run
+.Cm freebsd-update to upgrade a jail RELEASE to the specified
+RELEASE.
+.Pp
+Please note: basejails are upgraded by setting the jail's release
```

Comment:
The basejail updating stuff is not currently implemented, and may not be done the same way. This should be removed.